english/light-fire/magic-world/follow/dream/fight/fight.md

```diff
@@ -2,4 +2,6 @@ Looks like it was a bad dream. Good Morning!!
 
 What a relief!
 
+Apart from a splitting headache it appears that everything is normal. Just as you are about to breath a sigh of relief you notice that something seems to have been chewing on your pillows, and what is worse, that it may have been you. And why would they taste like marshmallows.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.